doc/source/cluster/config.rst

```diff
@@ -341,14 +341,14 @@ The key is the name of the node type, which is just for debugging purposes.
                 resources: {"CPU": 2}
                 min_workers: 0
                 max_workers: 0
-            ray.worker.small:
+            ray.worker.default:
                 node_config:
                   InstanceType: m5.large
                   InstanceMarketOptions:
                       MarketType: spot
                 resources: {"CPU": 2}
                 min_workers: 0
-                max_workers: 1
+                max_workers: 2
```

Comment:
maybe we should completely remove it if we are implying it from the global?

Reply:
Removed it so that it matches aws/defaults.yaml